### ISSUE TITLE
Testing the model 

### ISSUE THREAD

**Issue by abhigoku10** [Deep-Learning-Person-Re-Identification/Implementaion-1#42]
@Ning-Ding @prashanthbasani @wangbm  HI guys thanks for the wonderful work , just wanted to know after running the model how to test it , i donot see any test.py file can you let me know the mode to test it

**Comment by M-Ghorbel** [comment]
Hi
i'm looking for the testing code too

**Comment by bmiftah** [comment]
I guess the test file is not separate one, it is a function inside the main function  as in : - 

def test(model,val_or_test='test'):
    a,b = _get_test_data(val_or_test)     # from camera pairs
    return model.predict_on_batch([a,b])
.....
The data for test has been already set aside inside data_preparation file 
my guess is the test result needs to be written to file so that it can be easy to analyze or plot the curve

**Comment by M-Ghorbel** [comment]
Yes that's right but it's not giving the same cmc curve as the article.
